design/FY2018/Fan Energy Index.md

```diff
@@ -0,0 +1,250 @@
+
+ANSI/AMCA Standard 208-18 defines the calculation method for FEI. The standard has been written to support the adoption of energy efficient fans in the energy and construction codes and standards (e.g., ASHRAE Standards 90.1, 189.1, and California Appliances Standards Title 20) and to support utility rebate programs. FEI is a replacement for fan efficiency grade (FEG), a metric currently referenced in model energy codes and standards such as ASHRAE 90.1, ASHRAE 189.1, and IECC and in federal regulations in several Asian countries.
+
+Implementing FEI, based on the ANSI/AMCA standards 208-18 and 207-17, in EnergyPlus will enable its use to support: (1) building energy codes and standards, (2) mechanical engineers to evaluate and select energy efficient fans for various applications, and (3) utilities incentives/rebates programs for fans.
+
+## Fan Energy Index and Calculations 
+The following algorithms and calculation procedures (the SI unit version) are reproduced from the ANSI/AMCA standards 208-18: Calculation of the Fan Energy Index, and 207-17: Fan System Efficiency and Fan System Input Power Calculation.
+The fan energy index (FEI) is defined as a ratio of the electrical input power of a reference fan to the electrical input power of the actual fan for which the FEI is calculated, both calculated at the same duty point, i, which is characterized by a value of airflow (Q<sub>i</sub>) and pressure (P<sub>t,i</sub> or P<sub>s,i</sub>). FEI can be calculated for each point on a fan curve.
+
+$$FEI_{t,i}\;or\;FEI_{s,i}=\frac{Reference Fan Electrical Input Power}{Actual Fan Electrical Input Power} = \frac{FEP_{ref,i}}{FEP_{act,i}}$$ (1)
+
+Where,
+
+FEI<sub>t,i</sub>: FEI based on fan total pressure
+
+FEI<sub>s,i</sub>: FEI based on fan static pressure
+
+We propose to add field as a flag to the current fan model **Fan:SystemModel** for reporting FEI (considering that it can be substituted for **Fan:ConstantVolume**,**Fan:OnOff**, **Fan:VariableVolume**, and **FanPerformance:NightVentilation** objects and they may be deprecated and eventually removed in the future). We will use the user defined Fan objects to simulate the actual fan electrical input power FEP<sub>act,i</sub> at the design operating point. We will calculate the reference fan electrical input power FEP<sub>ref,i</sub> using the following method.
+
+Reference fan shaft power (H<sub>i,ref</sub>) is calculated on a fan total pressure basis or a static pressure basis, depending on the category of the reference fan to calculate FEI. However, in EnergyPlus **Fan:SystemModel**, only total pressure is defined. So the FEI is reportable only for limited category of fans that are calculated based on the fan total pressure, namely Centrifugal Housed, Centrifugal Inline, Centrifugal PRV Supply, Axial Inline, Laboratory Exhaust, Jet Fan and Circulating 
```

Comment:
Oh - I see the explanation here. There may be some confusion in the fan pressure terminology for the existing fan models. The "Design Pressure Rise" input for Fan:SystemModel (and the other simple fan models) is the change in pressure across the fan, and I'm 90% certain that is the fan *static* pressure, not total pressure. @EnergyArchmage @Myoldmopar @rraustad @larryscheier ?  The confusion grows with the description in the I/O Reference "This numeric field is the total system pressure rise experienced by the fan in Pascals at full flow rate...". And to make matters even more fun, there's ["total static pressure"](https://unmethours.com/question/14589/estimate-fan-pressure-rise-from-esp/) vs "external static pressure".  This [PNNL report](https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-18773.pdf) also implies that the EnergyPlus fan pressure input is TSP. Sorry to muddy the waters here at this late stage. Hopefully you or someone else can set me straight.

Regardless of which type of pressure the fan input represents, it would be nice to figure out a way to handle the other fan types.

Reply:
@mjwitte I did got confused about the documentation. I did some research and it seems this post https://unmethours.com/question/14589/estimate-fan-pressure-rise-from-esp/ concludes on the pressure is the total pressure.

No matter what type it is, I think without extra input it's hard to do the calculation/estimation from one to the other with the Fan:SystemModel. 

And probably we should make it clearer which pressure should the users input for the existing fan models. It seems to me that people make different assumptions. @Myoldmopar @EnergyArchmage @rraustad @larryscheier